src/applications/facility-locator/tests/components/SearchControls.unit.spec.jsx

```diff
@@ -17,6 +17,7 @@ describe('SearchControls', () => {
         .find('#facility-type-dropdown [selected]')
         .text(),
     ).to.equal('Choose a facility type');
+    expect(wrapper.find('.vads-u-color--secondary-dark').length).to.equal(2);
```

Comment:
A bit of a nitpick, but this test only confirms that there are 2 elements with the `.vads-u-color--secondary-dark` class. They could be the wrong ones. It would be even better to find the `label` elements, then assert that they have the correct class AND the "(*Required)" text.